tests/optimisation_test/optimisation_manager.py

```diff
@@ -15,22 +15,21 @@ def from_dict(cls, state_dict: dict={}):
         model.save()
         return cls(model)
     
-class OptimisationManager(OptimizerBase):
+class OptimisationManager():
     
     def __init__(self,
                  config_file: str,
                  batch_size: int = 15,) -> None:
-        self.config = self._read_config(file_path=config_file)
-        self.batch_size = batch_size
-        bound = self.generate_bound_from_config()
-        self.component_keys = bound.keys() 
+        self._config_dict = YamlHandler.load_config_file(config_file)
+        self._batch_size = batch_size
+        bound = self._generate_bound_from_config()
+        self._component_keys = bound.keys() 
         #self.target_name = None  
-
-        self.model = self.generate_model(pbounds=bound)
+        self.model = self._generate_model(pbounds=bound)
         self._probed_points = pd.DataFrame([])
 
     def construct_optimiser_from_config_file(self, config_file_path:str):
-        return State.from_dict(self.config['optimizer'])
+        return State.from_dict(self._config_dict['optimizer'])
```

Comment:
Rename State to something more appropriate like "OptimisationState" or such since this would clash with the Archemist global State